third-party-major.json

```
@@ -0,0 +1,87 @@
+{
+  "extends": [
+    ":prHourlyLimit4",
+    ":rebaseStalePrs",
+    ":renovatePrefix",
+    ":semanticCommits",
+    ":timezone(Australia/Melbourne)",
+    ":updateNotScheduled",
+    "preview:buildkite",
+    "preview:dockerCompose",
+    "docker:disableMajor"
+  ],
+  "packageRules": [
+    {
+      "depTypeList": ["dependencies"],
+      "semanticCommitType": "fix"
+    },
+    {
+      "commitMessageExtra": "{{newValue}}",
+      "commitMessageTopic": "{{depName}}",
+      "matchManagers": ["buildkite", "gomod", "npm", "nvm"]
+    },
+    {
+      "matchManagers": ["npm"],
+      "matchPackageNames": ["^@types/"],
+      "matchUpdateTypes": ["major"],
+      "enabled": true
+    },
```

I actually wonder if we need the inverse here; something like

```suggestion
    {
      "matchManagers": ["npm"],
      "matchUpdateTypes": ["patch", "minor"],
      "enabled": false
    },
```